zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/TestSupport.java

```diff
@@ -107,7 +107,8 @@ static Stream<ValueType> provideValueTypes() {
             ValueType.ROLE,
             ValueType.TENANT,
             ValueType.GROUP,
-            ValueType.MAPPING);
+            ValueType.MAPPING,
```

Comment:
❓ Can you remove MAPPING from here ? I just forgot to remove it when I implemented the exporters

Reply:
@nicpuppa I'm confused here. This is regarding the ES/OS exporters, not the CamundaExporter. We don't export mappings to the ES/OS exporters, right?

Reply:
Yeah, you are right. Please ignore all my comments